dev-util/coverity/coverity-2017.07.ebuild

```diff
@@ -0,0 +1,96 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Coverity provides the results of analysis on open source coding projects"
```

Comment:
Don't repeat the package name. Also please make it say what it does instead of what marketing said.

Reply:
Fixed